src/status_im2/contexts/emoji_picker/view.cljs

```diff
@@ -0,0 +1,200 @@
+
+(defn- footer
+  [theme active-category scroll-ref]
+  (let [on-press #(on-press-category %1 %2 active-category scroll-ref)]
+    (fn []
+      [rn/view {:style style/category-container}
+       [blur/view
+        {:style         style/category-blur-container
+         :blur-radius   (if platform/android? 20 10)
+         :blur-amount   (if platform/ios? 20 10)
+         :blur-type     (quo.theme/theme-value (if platform/ios? :light :xlight) :dark theme)
+         :overlay-color (quo.theme/theme-value colors/white-70-blur colors/neutral-95-opa-70-blur theme)}
+        [quo/showcase-nav
+         {:state     :scroll
+          :active-id @active-category
+          :data      emoji-picker.data/list-data
+          :on-press  on-press}]]])))
+
+(defn- clear
+  [active-category filtered-data search-text]
+  (reset! active-category :people)
+  (reset! filtered-data nil)
+  (reset! search-text ""))
+
+(defn- view-internal
+  [_]
+  (let [{:keys [on-select]}       (rf/sub [:get-screen-params])
+        scroll-ref                (atom nil)
+        set-scroll-ref            #(reset! scroll-ref %)
+        search-text               (reagent/atom "")
+        filtered-data             (reagent/atom nil)
+        active-category           (reagent/atom :people)
+        search-emojis             (debounce/debounce
+                                   (fn []
+                                     (when (pos? (count @search-text))
+                                       (let [results         (emoji-picker.utils/search-emoji
+                                                              @search-text)
+                                             has-results?    (not-empty results)
+                                             filtered-search (into []
+                                                                   (when has-results?
+                                                                     results))]
+                                         (reset! filtered-data filtered-search))))
+                                   constants/search-debounce-ms)
+        on-change-text            (fn [text]
+                                    (if (= text "")
+                                      (clear active-category filtered-data search-text)
+                                      (do
+                                        (reset! search-text text)
+                                        (search-emojis))))
+        on-clear-text             (fn []
+                                    (clear active-category filtered-data search-text))
+        on-viewable-items-changed (fn [event]
+                                    (handle-on-viewable-items-changed event
+                                                                      active-category
+                                                                      filtered-data))]
+    (fn [{:keys [theme] :as sheet-opts}]
+      (let [is-search-active? (pos? (count @search-text))]
```

Comment:
Same as another comment I made, the `is-prefix` is considered redundant with the question mark.